botorch/acquisition/input_constructors.py

```diff
@@ -457,6 +411,44 @@ def construct_inputs_mc_base(
     }
 
 
+@acqf_input_constructor(qSimpleRegret)
+def construct_inputs_mc_base(
+    model: Model,
+    training_data: MaybeDict[SupervisedDataset],
+    objective: Optional[MCAcquisitionObjective] = None,
+    posterior_transform: Optional[PosteriorTransform] = None,
+    X_pending: Optional[Tensor] = None,
+    sampler: Optional[MCSampler] = None,
+    **kwargs: Any,
+) -> Dict[str, Any]:
+    r"""Construct kwargs for qSimpleRegret. This function has a misleading
+    name.
+
+    Args:
+        model: The model to be used in the acquisition function.
+        training_data: Dataset(s) used to train the model. Not used.
+        objective: The objective to be used in the acquisition function.
+        posterior_transform: The posterior transform to be used in the
+            acquisition function.
+        X_pending: A `batch_shape, m x d`-dim Tensor of `m` design points
+            that have points that have been submitted for function evaluation
+            but have not yet been evaluated.
+        sampler: The sampler used to draw base samples. If omitted, uses
+            the acquisition functions's default sampler.
+        kwargs: Not used.
+
+    Returns:
+        A dict mapping kwarg names of the constructor to values.
+    """
+    return _construct_inputs_mc_base(
+        model=model,
+        objective=objective,
+        posterior_transform=posterior_transform,
+        X_pending=X_pending,
+        sampler=sampler,
+    )
```

Comment:
Is the idea here to not give `construct_inputs_mc_base` dual purpose? So we construct a public method that only serves `qSimpleRegret` and use the private method for the common functionality. If so, should we rename the public method to `construct_inputs_qSimpleRegret` or similar to avoid "This function has a misleading name."?